sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -220,14 +220,14 @@ public InstrumentationLibraryInfo getInstrumentationLibraryInfo() {
   }
 
   /**
-   * Returns the end nano time (see {@link System#nanoTime()}). If the current {@code Span} is not
-   * ended then returns {@link Clock#nanoTime()}.
+   * Returns the end nano time (see {@link System#nanoTime()}) or zero if the current {@code Span}
+   * is not ended.
    *
    * @return the end nano time.
    */
   private long getEndEpochNanos() {
     synchronized (lock) {
-      return getEndNanoTimeInternal();
```

Comment:
Maybe we want to keep the behavior to expose the duration for active (for example in OpenCensus z-pages we want to show for how long the Span was active), we have two options:
* Add a hasEnded on the SpanData and keep the end-time to be Clock::now
* Expose the Clock instance in SpanData.

I tend to like more the first option.

Reply:
That comment would be better at #74. 😃

Actually I'd like to have the hasEnded on the ReadableSpan already.